src/integrationTest/java/uk/gov/hmcts/reform/sscs/servicebus/TopicJmsTest.java

```diff
@@ -30,7 +30,9 @@
         "clientId", "guest", "guest",
         "amqp://localhost:8899?amqp.idleTimeout=120000"
             + "&amqp.saslMechanisms=PLAIN&transport.trustAll=true"
-            + "&transport.verifyHost=false");
+            + "&transport.verifyHost=false",
```

Comment:
transport.verifyhost should be true, where possible - again you may need to switch this off for local testing against qpid broker-J but it should be true in production

Reply:
and transport.trustAll has the same effect as the dummy TrustManager, so that should be false in production

Reply:
Just realised this is test code - ignore me!